### ISSUE TITLE
Weird scrollbar behavior on OS X

### ISSUE THREAD

**Issue by thomasjo** [atom/atom#9269]
It almost seems as if the hide/show logic for the scrollbars is inverted somehow; initially when loading Atom, no scrollbar is visible (as expected), upon scrolling no bar shows up. Then when scrolling stops , the scrollbar suddenly appears even though it should now be hidden.

Steps to reproduce:
1. Build Atom from master (also present in beta).
2. Open some file that allows scrolling, but do not scroll. Verify that there's no scrollbar visible.
3. Begin scrolling. Most likely no scrollbar will appear, but when you stop scrolling, one will likely appear.
4. Scroll once more and the scrollbar (if visible) will most likely go away.
5. Repeat ad infinitum.

I have the `editor.scrollPastEnd` option set to `true`, and running OS X 10.11.1.

I'm also experiencing scenarios when it doesn't show up at all.
Here's a screen recording where I'm attempting to illustrate what's happening. It almost seems as if it's showing up every other time. Although it also seems almost random at times.

![weird_scrolling](https://cloud.githubusercontent.com/assets/3622/10704972/d055eea0-79da-11e5-8181-e810c96adf1b.gif)

In summary, a bit hard to explain, but easily reproduced. I'm checking stable now to see if the problem is present there as well. Either way, I'll probably start bisecting to figure out when this regression was introduced.

**Comment by thomasjo** [comment]
The problem is *not* present in stable (1.0.20-7229d1a).

**Comment by simurai** [comment]
Noticing the same. Tree-view and settings are ok, just the scrollbar in the editor has this issue.

**Comment by zcbenz** [comment]
I tested different versions of electron, and this bug started to happened since the update to electron v0.32.x, e.g. the one with Chrome 45 upgrade. It seems that some behavior in Chrome 45 has changed.

**Comment by thomasjo** [comment]
@zcbenz I bisected this earlier and tracked it down to the v0.33.1 bump in Atom (https://github.com/atom/atom/commit/9354b865dd9df0c383b9518b52db5db7cbb82b7f), which supports your findings.

**Comment by nathansobo** [comment]
This seems to be a Chrome bug. I created a reproduction example and opened an issue on their tracker: https://code.google.com/p/chromium/issues/detail?id=554722

**Comment by arturparkhisenko** [comment]
I also got a very strange scrollbar in my project, not too much files, also its **Atom 1.2.2** and Windows 7/8/10 (same bug on all of them).
Screenshot:
![scrollbars](https://cloud.githubusercontent.com/assets/139212/11267713/a1360516-8eb5-11e5-9d2c-9d83c815b22c.png)

**Comment by nathansobo** [comment]
This issue is [still in progress](https://bugs.chromium.org/p/chromium/issues/detail?id=535684) on the Chromium side and is officially slated for Chrome 49, which is currently in beta. I asked whether that estimate as accurate on the linked thread because it doesn't seem like it is. Seems like we may be waiting until Chrome 50 or later for a fix. Considering it's in progress, it doesn't seem high-priority enough to justify a hacky workaround, even if we could come up with one.

**Comment by nathansobo** [comment]
@lee-dohm Heads up: Marking this as blocked for now.

**Comment by nathansobo** [comment]
The patch has landed in Chromium and this will be fixed in Chrome 51.
